proxyfinder.go

```diff
@@ -31,6 +31,15 @@ func NewProxyFinder(pacurl string) *ProxyFinder {
 	return pf
 }
 
+func (pf *ProxyFinder) WrapHandler(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		if pf.fetcher != nil {
```

Comment:
Does `pf.fetcher` change after starting? If not, move this check outside `return http.HandlerFunc(…)`.

If it's the case in practice, but not guaranteed, consider guaranteeing it (or raising an issue to address this later).

Reply:
No it does not change. So I moved the check outside the HandlerFunc.

Done.

Reply:
Fixed in 82a1dd29ef1b88e614aa5e9c8559680c85401435. Will rebase-squash prior to merging.